database/index.js

```diff
@@ -1,7 +1,13 @@
 const mongoose = require('mongoose');
 const AutoIncrement = require('mongoose-sequence')(mongoose);
 
-mongoose.connect('mongodb://localhost/airbnbDesc', { useNewUrlParser: true });
+const mongoUri = process.env.MONGODB_URI || 'mongodb://localhost/airbnbDesc';
+
+console.log('PROCESS.ENV.MONGODB_URI', process.env.MONGODB_URI);
```

Comment:
Still need these console logs?

Reply:
Thanks! About to clone this to my EC2 instance and will be needing these console.logs :)